src/Phinx/Db/Adapter/SQLiteAdapter.php

```diff
@@ -1241,9 +1254,6 @@ protected function getColumnSqlDefinition(Column $column)
         if ($column->getPrecision() && $column->getScale()) {
             $def .= '(' . $column->getPrecision() . ',' . $column->getScale() . ')';
         }
-        if (($values = $column->getValues()) && is_array($values)) {
-            $def .= " CHECK({$this->quoteColumnName($column->getName())} IN ('" . implode("', '", $values) . "'))";
-        }
```

Comment:
What happened to the support for `CHECK`?

Reply:
The check constraint is the mentioned enum emulation.